packages/dapp-connector/src/manager.mocks.ts

```
@@ -25,8 +25,8 @@ export const managerMock: DappConnectorManager = {
   getWalletConnectorScript(): string {
     return ''
   },
-  handleEvent(): Promise<void> {
-    return Promise.resolve()
+  handleEvent(): void {
+    return undefined
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)